service/errors/errors.go

```diff
@@ -295,6 +296,13 @@ func NewDeploymentUnprocessableError() *Error {
 	}
 }
 
+func NewDeploymentNotFoundError() *Error {
```

Comment:
Error is not translated in the transport layer because it is never passed to transport layer (for now).